src/monster.h

```diff
@@ -303,8 +304,11 @@ class monster : public Creature, public JsonSerializer, public JsonDeserializer
         std::string unique_name; // If we're unique
         bool hallucination;
 
-        bool setpos(const int x, const int y, const bool level_change = false);
-        bool setpos(const point &p, const bool level_change = false);
+        // level_change == true means "monster isn't spawned yet, don't update position in tracker"
```

Comment:
So it does not actually have to be a level change? The parameter should probably be renamed to something more useful that doesn't require one to read the comment.

Actually: what's the difference between calling `setpos(..., true)` and `spawn(...)`? Both function set the position of the monster and neither updates the `Creature_tracker`. Why not remove the parameter completely and use `spawn` at the few places in game.cpp where this has been called with `level_change = true`?